articles/article-002-camel-case.md

```diff
@@ -0,0 +1,24 @@
+
+<!-- BEGIN TITLE -->
+# CamelCase, or why coding conventions matter
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+One problem that often arises when writing code is the naming of variables. Often multi-word names are used for variables but in most programming languages it is not possible to use spaces in a variable name. One programming style that tries to structure this is called CamelCase. It is the practice of writing each word in the middle of a phrase with a Capital letter. This greatly improves readability.
```

Comment:
Do you intentially have a the sentence "this greatly improves readability" twice (once in normal writing and once in camel case).

Reply:
Nope that was an accident.